buildSrc/src/main/kotlin/Crates.kt

```diff
@@ -4,7 +4,7 @@
  */
 
 object Crates {
```

Comment:
I think my brain would expect these to be lists if the object was named `CrateSet` or something like that

Reply:
I'm fine with `CrateSet` 👍